scripts/tests/chiptest/runner.py

```diff
@@ -118,29 +119,55 @@ def get(self):
         return self.queue.get()
 
 
-class Runner:
+@dataclass
+class SubprocessInfo:
+    kind: str
+    path: pathlib.Path | str
+    wrapper: tuple[str, ...] = ()
+    args: tuple[str, ...] = ()
+
+    def __post_init__(self):
+        self.path = pathlib.Path(self.path)
+
+    def __repr__(self) -> str:
```

Comment:
Why this is overwritten? The whole benefit of dataclass is that it provides `__init__` and `__repr__` automatically.